src/Conversion/KrnlToLLVM/KrnlCall.cpp

```diff
@@ -119,6 +119,13 @@ class KrnlCallOpLowering : public ConversionPattern {
       parameterTypeList.emplace_back(opaquePtrTy);
       parameterList.emplace_back(omTensor);
       omTensors.emplace_back(omTensor);
+    } else if (ty.isa<NoneType>()) {
+      // generate llvm null pinter for NoneType
```

Comment:
nit: `generate` -> `Generate`, and please end the sentence with a full stop.

Reply:
Fixed. Thanks.